system-x/services/ldap/src/main/java/software/tnb/ldap/validation/LDAPValidation.java

```diff
@@ -25,7 +25,7 @@ public LDAPValidation(LDAPConnectionPool ldapConnectionPool) {
 
     public List<SearchResultEntry> getEntries(String filter) {
         try {
-            SearchRequest searchRequest = new SearchRequest("dc=example,dc=org", SearchScope.SUB, filter);
+            SearchRequest searchRequest = new SearchRequest("dc=redhat,dc=com", SearchScope.SUB, filter);
```

Comment:
can we make this dynamic (I think `StringUtils.substringAfter(account().getUsername(), ",")` ) same as `OPENLDAP_ROOT_DN_SUFFIX` env variable for container

Reply:
That's possible retrieving info from the "lastBindRequest". Done.